ateam_kenobi/src/plays/halt_play.cpp

```diff
@@ -33,7 +33,7 @@ void HaltPlay::reset()
 }
 
 std::array<std::optional<ateam_msgs::msg::RobotMotionCommand>, 16> HaltPlay::runFrame(
-  const World & world)
+  const World &)
```

Comment:
did you mean to remove the reference to `world` here?

Reply:
I removed the parameter name because we don't use that parameter in this function and this is the normal way to suppress unused parameter warnings from the compiler.

Reply:
Ah sorry, forgot this inherits from BasePlay!